### PR TITLE
fix(message): send message to user before store

### PR DESCRIPTION
**Pull request by Jekrimo**
### What this PR does 📖
- Send message before storing

### Which issue(s) this PR fixes 🔨
- Resolve #
#4736 

### Special notes for reviewers 🗒️
I think this is right?

### CODE REVIEW

File: libraries/Iridium/chat/ChatManager.ts [hunk 1]
```diff
@@ -744,24 +747,20 @@ export default class ChatManager extends Emitter<ConversationMessage> {
         lastReadAt: Date.now(),
       },
     }
+
+    await iridium.connector.store(partial, {
```
Comment (aewing) [inline libraries/Iridium/chat/ChatManager.ts:751]: I think we can just call store and not await it for the UI's sake

Comment (aewing) [reply libraries/Iridium/chat/ChatManager.ts:751]: Maybe we don't need to await the publish either... we might want to add a .catch that marks the message as unsent locally though so we can offer the user to re-send

**Review by aewing — APPROVED** [review]
Feel free to merge but it would be cool to test not blocking by removing the awaits

File: libraries/Iridium/chat/ChatManager.ts (hunk 1 above)
Comment (Jekrimo) [reply libraries/Iridium/chat/ChatManager.ts]: Good call, fixeded

**Pull request merged**